### PR TITLE
phone validator

### PR DESCRIPTION
**Pull request by salman0ansari**
Added phone validator for Indian numbers.

### CODE REVIEW

File: validators/phone_IN.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const dry = require('drytypes')
+
+module.exports = dry.makeDryType((x) => {
+
+  if (typeof x !== 'string') {
+    return { success: false, message: 'Phone number should be string!' }
+}
+// const phoneRegex = /^(\+?91|0)?[6789]\d{9}$/;
+const phoneRegex = /(((^[\+,0][9][1])(((\s[0-9]{7,10})|(\S[0-9]{7,10}))|([-]\S[0-9]{7,10})))|((^[\+,0][2]{2,2})((\S[0-9]{7,8})|((([-])[0-9]{7,8})|(\s[0-9]{7,8})))))|(((^[6,7,8,9][0-9]{9,9}))|(^[0,\+](([9][1)|[6,7,8,9]))[0-9]{8,9}))/gm;
```
Comment (SwapnilSoni1999) [inline validators/phone_IN.js:9]: I wonder if we can manage to validate all countries' phone numbers withing single validator lib

**Review by SwapnilSoni1999 — CHANGES_REQUESTED** [review]
make the required changes

File: validators/phone_IN.js (hunk 1 above)
Comment (salman0ansari) [reply validators/phone_IN.js:9]: hmm that's a good idea

**Comment by salman0ansari** [comment]
closing for now

**Pull request closed without merging**